ql/src/test/org/apache/hadoop/hive/ql/txn/compactor/TestWorker.java

```diff
@@ -1180,19 +1180,19 @@ private void verifyTxn1IsAborted(int compactionNum, Table t, CompactionType type
   }
 
   // With high timeout, but fast run we should finish without a problem
-  @Test(timeout=1000)
+  @Test(timeout=2000)
```

Comment:
Just curious, why is this changed?

Reply:
In one of the builds this test failed with timeout. Could not reproduce it locally, but increased the value in order to prevent flakyness in the CI builds.